packages/realm/README.md

```diff
@@ -1,24 +1,27 @@
-<picture>
-    <source srcset="https://github.com/realm/realm-js/blob/HEAD/media/logo-dark.svg" media="(prefers-color-scheme: dark)" alt="realm by MongoDB">
-    <img src="https://github.com/realm/realm-js/blob/HEAD/media/logo.svg" alt="realm by MongoDB">
-</picture>
+> [!NOTE]
+> Realm is now Atlas Device SDK - [Learn More](https://www.mongodb.com/blog/post/realm-now-part-atlas-platform?utm_medium=github_atlas_CTA&utm_source=realm_js_github)
 
-# About Realm Database
+<img align="right" style="min-width:150px;width:30%;" src="https://raw.githubusercontent.com/realm/realm-js/main/media/atlas-device-sync.svg" alt="Atlas Device Sync">
 
-Realm is a mobile database that runs directly inside phones, tablets or wearables.
-This project hosts the JavaScript versions of [Realm](https://www.mongodb.com/docs/realm/). Currently, we support React Native (JSC & Hermes on iOS & Android), Node.js and Electron (on Windows, MacOS and Linux).
+# Atlas Device SDK
```

Comment:
I was contemplating putting "(for JavaScript & TypeScript)" here as well, but it felt a bit verbose in a title.